src/BinSkim.Rules/PERules/BA2004.EnableSecureSourceCodeHashing.cs

```diff
@@ -194,75 +200,153 @@ public void AnalyzeNativeBinaryAndPdb(BinaryAnalyzerContext context)
 
                     if (sf.HashType != HashType.SHA256)
                     {
+                        // Adding the `MD5` or `SHA1` etc, comment delimited by a backquote/backtick (`) to the library name
```

Comment:
>Adding [](http://example.com/codeflow?start=27&length=6)

Is this a useful comment? Where is the period at the end of your sentence? Please make sure our coding standard documents this style requirement, we should do this for every NIT code review note reviewers have to offer.

A better use of a comment here is not to provide a human-readable description of what the code is doing but to instead provide a readable example of the output of the code. IMO.